cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,114 @@
 describe('Web Tables page', () => {
-  it('', () => {
+    const userData = {
+      firstName: 'Anna',
+      lastName: 'Lymorenko',
+      email: 'anyalymorenko@gmail.com',
+      age: '23',
+      salary: '1000',
+      department: 'QA'
+    };
+    beforeEach(() => {
+      cy.visit('https://demoqa.com/webtables');
+    });
 
+    it('should have pagination', () => {
+      cy.get('.-pagination')
+        .should('contain', 'Previous')
+        .and('contain', 'Next');
+      cy.contains('.-pageInfo', 'Page')
+        .should('exist');
+    });
+
+    it('should have row count selection', () => {
+      cy.get('select')
+        .select('10 rows');
+    });
```

Comment:
Test case for row count selection should include assertions to verify the change in displayed rows count.